README.md

```diff
@@ -1 +1,320 @@
+
+```ballerina
+import ballerinax/awslambda;
+
+@awslambda:Function
+public function processImages(awslambda:Context ctx, json input) returns json|error {
+    return "OK";
+}
+```
+
+In Ballerina, any function can be exposed as an AWS Lambda function, by annotating a function which has a similar function signature as the above function with `awslambda:Function`. The `awslambda:Context` object contains contextual information on the current execution, such as the request id, tracing information, and the remaining execution time. The payload is passed as a `json` value in the `input` variable, and the function can return a `json` as a response, or else an `error` can be returned to signal an error situation to the Lambda runtime. 
+
+In the next step, you will be updating the code to extract the S3 object information retrieved by the Lambda function. Here, according to the input S3 event payload structure, we have extracted the bucket, object name and the version. 
+
+```ballerina
+import ballerina/system;
+import ballerinax/awslambda;
+
+@awslambda:Function
+public function processImages(awslambda:Context ctx, json input) returns json|error {
+   int count = input.Records.length();
+   int i = 0;
+   while (i < count) {
+       string bucket = <string> input.Records[i].s3.bucket.name;
+       string name = <string> input.Records[i].s3["object"].key;
+       string objVersion = <string> input.Records[i].s3["object"].versionId;
+       i = i + 1;
+   }
+   return "OK";
+}
+```
+
+The next step is to invoke the Amazon Rekognition service to examine the image for text, and if there is any, extract the text that is there in the image. Here, you can see how the Amazon Rekognition connector is initiazed with the user credentials and the optional Amazon region setting. 
+
+```ballerina
+import wso2/amazonrekn;
+import wso2/amazoncommons;
+import ballerina/system;
+import ballerinax/awslambda;
+
+amazonrekn:Configuration config = {
+    accessKey: system:getEnv("AMAZON_ACCESS_KEY"),
+    secretKey: system:getEnv("AMAZON_SECRET_KEY"),
+    region: "us-west-2"
+};
+
+amazonrekn:Client reknClient = new(config);
+
+@awslambda:Function
+public function processImages(awslambda:Context ctx, json input) returns json|error {
+   int count = input.Records.length();
+   int i = 0;
+   while (i < count) {
+       string bucket = <string> input.Records[i].s3.bucket.name;
+       string name = <string> input.Records[i].s3["object"].key;
+       string objVersion = <string> input.Records[i].s3["object"].versionId;
+       amazoncommons:S3Object obj = { bucket: bucket, name: name, objVersion: objVersion };
+       amazonrekn:Label[] labels = check reknClient->detectLabels(obj);
+       boolean text = false;
+       foreach amazonrekn:Label label in labels {
+           if (label.name == "Text") {
+               text = true;
```

Comment:
Cant we directly add `string result = check reknClient->detectText(obj);` here since we don't seem to do any additional processing of the result?

Might be applicable to https://github.com/ballerina-guides/ballerina-awslambda-deployment/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R175 as well.

Reply:
That's true, thanks for the suggestion.